CHANGELOG.md

```diff
@@ -9,14 +9,14 @@ Removed legacy code and dependencies.
 ### New
 * Support am instrument options in snippet client.
 * Support adb reverse in `AdbProxy`.
-* Improved mechaism for tracking repeat and retry records in test report.
+* Improved mechanism for tracking repeat and retry records in test report.
 
 ### Breaking Changes
```

Comment:
Should we mention python min version bump?

Reply:
Isn't it already mentioned?